EnglishFilter/sections/general_extensions.txt

```diff
@@ -3081,9 +3083,10 @@ saavn.com#%#!function(a,b){b=new MutationObserver(function(){a.classList.contain
 ! https://github.com/AdguardTeam/AdguardFilters/issues/5388
 csgoconfigs.com#%#document.cookie = "AD_SHOWN=true";
 ! https://forum.adguard.com/index.php?threads/littlebyte-net.22689/
-littlebyte.net#%#AG_onLoad(function(){AG_each('a[onclick^="getActionContent"]',function(a){a.setAttribute('href','#');a.removeAttribute('target');})})
+littlebyte.net#%#AG_onLoad(function(){AG_each('a[onclick^="getActionContent"]',function(a){a.setAttribute('href','#');})})
+littlebyte.net#%#//scriptlet("remove-attr", "target   ", "a[onclick^='getActionContent']")
```

Comment:
`littlebyte.net#%#//scriptlet('remove-attr', 'target', 'a[onclick^="getActionContent"]')`. Is this rule necessary?

Reply:
Issue is not found on our forum and `littlebyte.net` main page gives no acces to other pages, so i am not able to test this one.